src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+}
+
+.navlink {
+  display: flex;
+  height: 60px;
+  align-items: center;
+}
+
+ul {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0;
+}
+
+li {
+  margin-left: 20px;
```

Comment:
better use this:

```
.nav_item:not(:last-child) {
  margin-right: 20px
}
```

**`There should not be margins before the first and after the last list items`**